README.md

```diff
@@ -233,6 +233,14 @@ For instance:
 If you use `target.merkle.tree.file.name` it will render the string
 `foobar` and `target.proof.hash` will render `abcd`.
 
+### How to specify the public root URL for public assets?
+
+If you're not using create-react-app as suggested in the [Installation](#installation) section, you can build a static, production-ready version of the admin using a custom public asset path root by specifying the `KINTO_ADMIN_PUBLIC_PATH` environment variable and running the `npm run build` command:
+
+```
+$ KINTO_ADMIN_PUBLIC_PATH='/admin/' npm run build
```

Comment:
Does it means that we will have the signoff plugin activated in that case? What about the max item per page?